consensus/ibft/pos.go

```diff
@@ -146,7 +146,7 @@ func (pos *PoSMechanism) preStateCommitHook(rawParams interface{}) error {
 	}
 
 	// Deploy Staking contract
-	contractState, err := stakingHelper.PredeployStakingSC(nil)
+	contractState, err := stakingHelper.PredeployStakingSC(nil, stakingHelper.PredeployParams{})
```

Comment:
@Aleksao998 You will need to fill in the fields of PredeployParams

Reply:
Fixed 703cb2ccaac20031a5f1addc0c4df1f79506659d